Sources/SwiftDocC/Semantics/Metadata/Metadata.swift

```diff
@@ -87,6 +90,7 @@ public final class Metadata: Semantic, AutomaticDirectiveConvertible {
         "supportedLanguages"    : \Metadata._supportedLanguages,
         "_pageColor"            : \Metadata.__pageColor,
         "titleHeading"          : \Metadata._titleHeading,
+        "redirects"              : \Metadata._redirects,
```

Comment:
Apologies for the super nit but the misalignment bugged me

```suggestion
        "redirects"             : \Metadata._redirects,
```

Reply:
ha ha no problem - updated